actions/appConfig.js

```diff
@@ -56,7 +56,7 @@ class AppConfig {
         payload.enableDelete = params.enableDelete;
 
         // These are from configs and not activation related
-        this.configMap.fgSite = params.fgSite;
+        this.configMap.fgSite = params.spSite || params.fgSite;
```

Comment:
We might need to set the spSite in the payload as-well.
Since internal activation calls mostly only pass over the payload and configs are used from env.

Reply:
Good catch, thanks @raga-adbe-gh!
Updated the PR.